CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 This file is used to list changes made in each version of the resolver cookbook.
 
+## Unreleased
+
+- Sous Chefs Adoption
+- Add inspec Tests
+- Remove Deprecated Platforms
+- Change ci to point to Sous Chefs
```

Comment:
Remove this as it's a part of `Sous Chefs Adoption`.

Reply:
Please remove this as I mentioned.

Reply:
Im not sure what happened here. I made all these changes, but they didn't push, when I ran diff on my local branch it showed no differences but I force pushed and all my changes finally showed up.